examples/rgw/boto3/service-2.sdk-extras.json

```diff
@@ -46,6 +46,37 @@
                 }
             }
         },
+        "ReplicationRule":{
+            "members":{
+		"Source": {
+		    "shape":"S3RepSource",
+		    "documentation":"Replication source zone.",
+		    "locationName":"Source"
+		}
+	    }
+	},
+	"S3RepSource": {
+	    "type": "structure",
+	    "members": {
+                "Zone": {
+                    "shape":"Zone",
+                    "documentation":"<p>Replication source zone.</p>",
+                    "location":"querystring",
+                    "locationName":"Zone"
```

Comment:
these aren't in the query string, so we don't need the "location". and i think "locationName" can be omitted if it matches the name of the member

Reply:
Thanks..Done.